ow_faults/include/ow_faults/FaultInjector.h

```diff
@@ -68,9 +68,19 @@ class FaultInjector
 private:
   float powerTemperatureOverloadValue;
   
+
+
+
+  float faultPanValue;
+  float faultTiltValue;
```

Comment:
These variables need to be prefixed with "m_" per our coding standard for member variables.  I was confused seeing them in the .cpp file, not knowing where they were defined.